lib/jnpr/junos/utils/sw.py

```diff
@@ -241,7 +243,9 @@ def pkgaddISSU(self, remote_package, **kvargs):
 
     def _parse_pkgadd_response(self, rsp):
         got = rsp.getparent()
-        rc = int(got.findtext('package-result').strip())
+        # If <package-result> is not present, then assume success.
+        # That is, assume <package-result>0</package-result>
```

Comment:
I can't guarantee it's a perfect assumption, but I do believe it's valid. In my experience, platforms which do not include `<package-result>` will include an `<rpc-error>` element if there's an error. We'll raise an `RpcError` exception in that case.